evap/staff/forms.py

```diff
@@ -204,6 +213,14 @@ def validate_unique(self):
         except forms.ValidationError as e:
             self._update_errors(e)
 
+    def save(self, *args, **kwargs):
+        responsibility = self.cleaned_data['responsibility']
+        self.instance.responsible = responsibility == Contribution.IS_RESPONSIBLE
+        self.instance.can_edit = (responsibility == Contribution.IS_RESPONSIBLE) or (responsibility == Contribution.IS_EDITOR)
+        if responsibility == Contribution.IS_RESPONSIBLE:
+            self.instance.comment_visibility = Contribution.ALL_COMMENTS
```

Comment:
please put `responsibility == Contribution.IS_RESPONSIBLE` and `responsibility == Contribution.IS_EDITOR` into a variable